packages/app/src/firebaseServerApp.ts

```diff
@@ -67,6 +93,16 @@ export class FirebaseServerAppImpl
       ...serverConfig
     };
 
+    // Ensure that the current time is within the authIdtoken window of validity.
```

Comment:
authIdtoken and appChecktoken look like literals that should be backticked.

Reply:
Fixed.